### ISSUE TITLE
[a11y] "Requested desktop version" toast is not announced

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2921]
## Steps to reproduce
- enable voiceview
- Visit a site
- Click "Request desktop site"

### Expected behavior
"Requested desktop version" toast is announced, followed by "WebView" and site specific content.

### Actual behavior
Toast is not announced: "WebView" is immediately spoken.

Because the tooltip is so clear, I'm not too concerned about this, at least for english users.

### Device information
* Fire TV device: ?
* Latest affected Firefox version: v4.5 master 2cc87063e1808eaaa291b818cb22c952c9469607
* Earliest affected Firefox version: ?
* Turbo Mode: ?
